cmd/devstream/install.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var installCMD = &cobra.Command{
-	Use:   "install",
+	Use:   "apply",
 	Short: "Install tools defined in config file",
 	Long:  `Install tools defined in config file`,
```

Comment:
For the short and long descriptions, I think we need a better one for the short and long descriptions for the short and long descriptions.

Example:

Short: Creates or updates DevOps tools according to DevStream configuration file.

Long: Creates or updates DevOps tools according to DevStream configuration file. DevStream will generate and execute a new plan based on the config file and the state file by default.

Reply:
Please also make sure "install" is changed to "apply" in all places, such as logging, etc. A grep might help.

Reply:
> Please also make sure "install" is changed to "apply" in all places, such as logging, etc. A grep might help.

these logs may not be modified:
![image](https://user-images.githubusercontent.com/94163234/148036096-36a75ec0-765d-4e37-b851-006c0a01b61c.png)

Reply:
> For the short and long descriptions, I think we need a better one for the short and long descriptions for the short and long descriptions.
> 
> Example:
> 
> Short: Creates or updates DevOps tools according to DevStream configuration file.
> 
> Long: Creates or updates DevOps tools according to DevStream configuration file. DevStream will generate and execute a new plan based on the config file and the state file by default.

modified